NEWS.md

```diff
@@ -4,6 +4,9 @@
 * Introduces `REPLICATION_FACTOR` environment variable for configuring Kafka topic replication (MODINVSTOR-694)
 * Provides `instance-storage 7.7`
 
+* Provides `holdings-storage 4.6` (MODINVSTOR-669)
+* Add effective holdings location attribute to holdings record (MODINVSTOR-669)
```

Comment:
I tend to follow the convention that the provides statements are less important than the functional changes.

Please re-order these statements and include them in their own section separate to the previous release.

```suggestion
* Adds `effective location` attribute to holdings record (MODINVSTOR-669)
* Provides `holdings-storage 4.6` (MODINVSTOR-669)
```